### PR TITLE
docs: Adding MQTT instructions and updating influxdb dashboard screen…

### PR DESCRIPTION
**Pull request by antoniomtz**
…shot

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

### CODE REVIEW

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 1]
```diff
@@ -58,13 +58,15 @@ You can update the object detection environment variables in file: `configs/open
...
 
 ## Build and Run Pipeline
 
 1. Build the python app and profile-launcher: `make build-python-apps`
 2. Download sample video files: `cd benchmark-scripts/ && ./download_sample_videos.sh && cd ..`
 3. Start simulator camera if not started: `make run-camera-simulator`
-4. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`
-5. To stop the running pipelines: `make clean-profile-launcher` to stop and clean up the client side containers, or `make clean-all` to stop and clean up everything.
\ No newline at end of file
+4. Run MQTT broker: `docker run --network host --rm -d -it -p 1883:1883 -p 9001:9001 eclipse-mosquitto`
```
Comment (jim-wang-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:69]:
```suggestion
4. (Optional) Run MQTT broker: `docker run --network host --rm -d -it -p 1883:1883 -p 9001:9001 eclipse-mosquitto`
```

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 2]
```diff
@@ -58,13 +58,15 @@ You can update the object detection environment variables in file: `configs/open
...
+5. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 MQTT=127.0.0.1:1883 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`
```
Comment (jim-wang-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:70]:
```suggestion
5. To start object detection pipeline: `PIPELINE_PROFILE="object_detection" RENDER_MODE=1 MQTT=127.0.0.1:1883 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --workload ovms`  (remove the MQTT environment variable if not using it)
```

File: docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md [hunk 3]
```diff
@@ -58,13 +58,15 @@ You can update the object detection environment variables in file: `configs/open
...
+6. Use the container name as the MQTT topic to subscribe to the inference metadata. Do a `docker ps` to know the container name.
```
Comment (jim-wang-intel) [inline docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md:71]:
```suggestion
6. If do use MQTT, use the container name as the MQTT topic to subscribe to the inference metadata. Do a `docker ps` to know the container name.
```

**Review by jim-wang-intel — COMMENTED** [review]
some suggestion to change regarding MQTT optional

**Review by brian-intel — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**